unitary/examples/fox_in_a_hole/fox_in_a_hole.py

```diff
@@ -0,0 +1,248 @@
+            move_str = self.take_random_move()
+            self.history_append_move(move_str)
+            self.history_append_state()
+            step_nr+=1
+        if step_nr==max_step_nr:
+            print("\nIt seems you have lost :-(. Try again.\n")
+        self.print_history()
+
+    def print_history(self):
+        """Prints out the history of states, guesses and moves."""
+        print('History:')
+        for hist_elem in self.history:
+            print(hist_elem)
+
+
+class ClassicalGame(Game):
+    """Classical Fox-in-a-hole game."""
+
+    def initialize_state(self):
+        self.state = self.hole_nr*[0.0]
+        index = self.rng.integers(low=0, high=self.hole_nr)
+        self.state[index] = 1.0
+
+    def state_to_string(self):
+        return str(self.state)
+
+    def check_guess(self,guess):
+        return self.state[guess] == 1.0
+
+    def take_random_move(self):
+        source = self.state.index(1.0)
+        direction = self.rng.integers(low=0, high=2)*2-1
+        if source==0 and direction==-1:
+            direction=1
+        elif source==self.hole_nr-1 and direction==1:
+            direction=-1
+        self.state[source]=0.0
+        self.state[source+direction]=1.0
+        if direction==-1:
+            dir_str = 'left'
+        else:
+            dir_str = 'right'
+        move_str = "Moving {} from position {}.".format(dir_str,source)
+        return move_str
+
+class Hole(enum.Enum):
+    """Enum for quantom object Hole."""
+    EMPTY = 0
+    FOX = 1
+
+class QuantumGame(Game):
+    """Fox-in-a-hole game with quantum effects.
+
+    Structure of state:
+    (QuantumWorld, [QuantumObject]) -> quantum world, list of holes
+    """
+
+    def __init__(self, hole_nr=5, iswap=False):
+        self.iswap = iswap
+        super().__init__(hole_nr=hole_nr)
+
+    def initialize_state(self):
+        index = self.rng.integers(low=0, high=self.hole_nr)
+        holes = []
+        for i in range(self.hole_nr):
+            hole = alpha.QuantumObject(
+                "Hole-{}-{}".format(i,i),
+                Hole.FOX if i==index else Hole.EMPTY)
+            holes.append(hole)
+        self.state = (alpha.QuantumWorld(holes), holes)
+
+    def get_probabilities(self):
+        """Gives back the probabilites for each hole."""
+        count=100
+        probs = self.hole_nr * [0.0]
+        peek_result = self.state[0].peek(objects=self.state[1],
+                                         convert_to_enum=False,
+                                         count=count)
+        for j in range(self.hole_nr):
+            for i in range(count):
+                probs[j]+=peek_result[i][j][0]
+        for j in range(self.hole_nr):
+            probs[j]=probs[j]/count
+        return probs
+
+    def state_to_string(self):
+        return str(self.get_probabilities())
+
+    def check_guess(self,guess):
+        measurement = self.state[0].pop([self.state[1][guess]])[0]
+        new_hole = alpha.QuantumObject(
```

Comment:
Can you put a TODO here (and elsewhere if other changes are needed) so that I or someone else can go and fix this once https://github.com/quantumlib/unitary/issues/21 is fixed?

Reply:
I put this todo into the code:
``` 
        # TODO: After Issue #21 is solved, there is no need for creating a new qobject
        # after each measurement.
```

Is there a standard way to refer to an issue?